homeassistant/components/stream/__init__.py

```diff
@@ -181,7 +184,9 @@ def start(self):
                 args=(self.hass, self, self._thread_quit),
             )
             self._thread.start()
-            _LOGGER.info("Started stream: %s", self.source)
+            _LOGGER.info(
+                "Started stream: %s", STREAM_SOURCE_RE.sub("//", str(self.source))
```

Comment:
Do we want to show that information has been redacted?

Reply:
We could do that, but I don't think that the fact that it is redacted has any importance. Having the username + password or maybe just the username would add more information, but aside from that just having the rest of the url is enough to identify the camera which is probably the point of the log message.